db/migrate/20220713195433_add_show_api_key_generation_view_to_spree_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddShowApiKeyGenerationViewToSpreeUser < ActiveRecord::Migration[6.1]
+  def change
+    add_column :spree_users, :show_api_key_view, :boolean, null: false, default: true
```

Comment:
I think that we want this disabled by default. @lin-d-hop?

Reply:
Yes please. Thanks for clocking this and checking @mkllnk